.gitignore

```diff
@@ -31,7 +31,7 @@ doc/_build/
 doc/locale/
 tests/.coverage
 tests/build/
-tests/test-server.lock
+# tests/test-server.lock  # no-longer required; you can remove this file from your sphinx directory
```

Comment:
I think it's best if you leave it for now since people might have it and could use `git add .`. Instead, you could add a CHANGELOG entry saying "you can remove it" and we would remove it during a cleanup? (or leave it indefinitely, it does not reall matter I think).